keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/EnvironmentSummary.kt

```diff
@@ -11,6 +12,7 @@ import com.netflix.spinnaker.keel.persistence.PromotionStatus
 /**
  * Summarized data about a specific environment, mostly for use by the UI.
  */
+@JsonPropertyOrder(value = ["name"])
```

Comment:
@erikmunson This now ensures that `name` will appear first in the environment summary properties.